isis/tests/Fixtures.cpp

```diff
@@ -1694,13 +1694,13 @@ namespace Isis {
     }
   }
 
-void NearMsiCameraCube::Init() {
+  void NearMsiCameraCube::SetUp() {
     TempTestingFiles::SetUp();
+    QString msicube = "data/near/msicamera/m0155881376f3_2p_cif_dbl.cub";
+    testCube.reset( new Cube(msicube, "r") ) ;
+  }
 
-    QString msicube = "m0155881376f3_2p_cif_dbl.cub";
-    QString testPath = tempDir.path() + "/" + msicube;
-    QFile::copy("data/near/msicamera/"+msicube, testPath);
-    testCube = new Cube(testPath, "rw");
-
+  void NearMsiCameraCube::TearDown() {
```

Comment:
I was uncertain about that because the Cube object will be deleted at the end of the test. What happens in TearDown()? Does the temp directory/files get removed at that point? If so, letting the pointer go out of scope when the class is destroyed seems too late to delete the Cube pointer, which I see most TearDown() methods doing.

Reply:
Yeah the parent TearDown for the test fixture will get called which removes the temp directory. It should be okay to just allow the smart pointer to clean up at that point; I don't think the cube file needs to still exist when the cube is destroyed.

Reply:
Ok, I see the difference from other fixtures. Thanks.